tools/perf/example.properties

```diff
@@ -586,7 +586,6 @@ build-id=Build Identifier
 update-channel=Update channel
 last-updated=Last Updated
 more-info=More Information
-gitInfo=Git commit info
```

Comment:
because it's a duplicated entity and now the parser errors on it :) I'll update both example.properties and example.l20n to use current settings' resource in a separate patch.